org_status/encoders/gitman.py

```diff
@@ -0,0 +1,20 @@
+import yaml
+from furl import furl
```

Comment:
Can you use `giturlparse`, we will be better off using parsers specifically designed to handle git urls.